tools/post-potd/post-potd.py

```diff
@@ -0,0 +1,37 @@
+from discord import SyncWebhook
+
+import json
+import requests
+
+from bs4 import BeautifulSoup
```

Comment:
This should be at the top

Reply:
What does the official style guide say? I was thinking that specific imports (i.e. `from`s) should go after any general ones, but good point about them not being grouped. In other words I'd probably move the `from discord` to be here rather than moving this at the top. But curious if you know of an official guideline.

Reply:
Yes, you are correct. The offical [guideline](https://peps.python.org/pep-0008/#imports) says 
"Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports"